.github/workflows/zephyr.yml

```diff
@@ -82,6 +82,8 @@ jobs:
         platforms: [
           {platform: tgl,
            real_cc: xtensa-intel_s1000_zephyr-elf/bin/xtensa-intel_s1000_zephyr-elf-gcc},
+          {platform: mtl,
+           real_cc: xtensa-intel_s1000_zephyr-elf/bin/xtensa-intel_s1000_zephyr-elf-gcc},
```

Comment:
This is the wrong compiler. We need
https://github.com/thesofproject/crosstool-ng/releases/download/gcc11.2/xtensa-lx7hifi4-elf.tar.bz2
in fact we should really update GCC to v11 (for the better front end checking. Not immediately urgent though).
GCC11 is here
https://github.com/thesofproject/crosstool-ng/releases/tag/gcc11.2

Reply:
@marc-hb you mean whether it matters for sparse which compiler version we use? Don't think so, as long as it can build the project

Reply:
Thanks @lyakh . Then I can add this back with a comment that it's temporary and not the "correct" compiler for MTL but it is the expected one right now and does not matter for sparse as long as there is one?

Maybe in another PR though because this one is urgently needed to fix the sparse build on ANY platform.

Reply:
@nashif fyi - the MTL GCC version is different than TGL (aka S1000 used in Zephyr SDK). Zephyr SDK will need an update. See above for link to binaries.

Reply:
MTL sparse submitted in new #6389